theme/styles/snippets/cart.scss

```diff
@@ -33,12 +34,30 @@ $cart-header-offset: 1px;
 
   &__inner {
     height: calc(100% - #{$nav-height});
-    padding-bottom: $cart-checkout-container-height;
+
+    &__items {
+      .CartItem:last-of-type {
+        margin-bottom: 0;
+      }
+    }
+    
+    &:after {
+      content: "";
+      display: block;
+      height: calc(#{$cart-checkout-container-height} - #{$cart-item-offset});
+      width: 100%;
+    }
```

Comment:
using a pseudo element because the `padding-bottom` that was here previously, doesn't seem to work when you have `scroll-overflow-y`.

Reply:
love them psuedos